src/common/variables.c

```diff
@@ -351,42 +315,48 @@ gchar *dt_variables_get_result(dt_variables_params_t *params)
 
 gboolean dt_variables_expand(dt_variables_params_t *params, gchar *string, gboolean iterate)
 {
-  gchar *variable = g_malloc(128);
-  gchar *value = g_malloc_n(1024, sizeof(gchar));
-  gchar *token = NULL;
+	gchar *variable = g_malloc(128);
+	gchar *value = g_malloc_n(1024, sizeof(gchar));
```

Comment:
Since `  ` (2 spaces) were replaced with `	` (tab), i'm not sure what is actually changed here...